README.md

```diff
@@ -7,6 +7,44 @@ In addition, a persistent instance will be available on GCP for non-technical us
 
 Dependency management is handled through Poetry - on first use, run `poetry install` to initialise the required packages.
 
+To get started interfacing with the GC Registry container, follow these steps to get started, using your IDE and API messenger of choice:
+- Make sure you have the Docker daemon running, either through Docker desktop or the CLI.
+- On first use, from the root directory run `docker compose -f compose.override.yml up --build` to build the image and run the container cluster. The cluster is composed of four containers:
+    - `gc_registry` - the FastAPI app and backend services for the registry
+    - `eventsotre` - an open-source streaming database that sequentially records registry events immutably
+    - `postgres_db_write` - a Postgres database container using the official image that receives write operations from the registry
+    - `postgres_db_read` - the parallel read Postgres instance that is written to simultaneously to the write instance, in order to optimise the concurrent processing of read operations without being detrimental to the performance of write operations.
+- By default, the container port mappings are: 
+    - The `gc_registry` FastAPI app - `8000`
+    - The `eventstore` instance GUI - `2113`
+    - The Postgres write intance - `5436`
+    - The Postgres read instance - `5438` 
+- Documentation comes in two flavours:
+    - `localhost:8000/docs` - Swagger-style schema and endpoint descriptions that allows basic interaction with API
+    - `localhost:8000/redoc` - alternative, non-interactive documentation from Redocly that is more human-readable than Swagger
+- This repository uses a [makefile](Makefile) for quick access to convenience functions that are common across both users and developers. For instance, `make dev` will quickly call Docker compose to load up the container cluster. 
+- Initially, the database instances will contain no elements. To get started quickly, we recommend seeding the database with some example User, Account, Device, and GC Bundle entities courtesy of Elexon (the UK aggregator of electricity system data) with the command `make db.seed`.
```

Comment:
Can you add a point here on running migrations. They will need that before seeding.